### PR TITLE
TDQ-15253 The four Word-based pattern indicator were not translated t…

### PR DESCRIPTION
**Pull request by xqliu-talend**
…o chinese when set the system language to chinese.

### CODE REVIEW

File: i18n/plugins/net.sourceforge.sqlexplorer.nl/src/net/sourceforge/sqlexplorer/messages_hr.properties [hunk 1]
```diff
@@ -74,7 +72,13 @@ DatabaseDetailView.Tab.Info.status=Info za
 DatabaseDetailView.Tab.Info.Property=Svojstvo
 DatabaseDetailView.Tab.Info.Name=Naziv tablice
 DatabaseDetailView.Tab.Info.Catalog=Katalog
-DatabaseDetailView.Tab.Info.Schema=\n\n	\n\nCollection\n\nSchema
+DatabaseDetailView.Tab.Info.Schema=
```
Comment (scorreia) [inline i18n/plugins/net.sourceforge.sqlexplorer.nl/src/net/sourceforge/sqlexplorer/messages_hr.properties]: bad formatting here.

File: i18n/plugins/org.talend.cwm.compare.ui.nl/src/messages_fr.properties [hunk 2]
```diff
@@ -36,7 +36,12 @@ RenameComparedElementAction.NoNewElement=Aucun 
 RefreshActionDelegate.errorUnableCreateNewCon=Impossible de cr�er une nouvelle connexion : {0}
 ReloadDatabaseAction.ConfirmMessageTitle=Confirmer le rechargement
 ReloadDatabaseAction.ConfirmMessage0=Cette action va recharger les m�tadonn�es de la connexion distante. Cela peut impacter les Jobs et analyses existants. Continuer ?
-ReloadDatabaseAction.ConfirmMessage1=\n\n	\n\n\nCette action va recharger les m�tadonn�es de la connexion distante. Cela peut impacter les analyses existantes. Continuer ?
+ReloadDatabaseAction.ConfirmMessage1=
```
Comment (scorreia) [inline i18n/plugins/org.talend.cwm.compare.ui.nl/src/messages_fr.properties]: bad formatting here.

File: i18n/plugins/org.talend.cwm.management.nl/src/main/java/org/talend/cwm/management/i18n/messages_zh_CN.properties [hunk 3]
```diff
@@ -135,42 +134,11 @@ EIndicatorChartType.EastAsiaPatternLowFrequencyStatistics=\u4E1C\u4E9A\u6A21\u5F
...
 DQRepositoryViewLabelProvider.ContextFolderName=\u4E0A\u4E0B\u6587
-RepositoryNodeHelper.TDQ_Libraries=\u5E93
-RepositoryNodeHelper.TDQ_Data_Profiling=\u6570\u636E\u5256\u6790
-RepositoryNodeHelper.context=\u4E0A\u4E0B\u6587
-RepositoryNodeHelper.metadata=\u5143\u6570\u636E
-RepositoryNodeHelper.fileDelimited=\u5B9A\u754C\u6587\u4EF6
-RepositoryNodeHelper.connections=\u6570\u636E\u5E93\u8FDE\u63A5
-RepositoryNodeHelper.Analyses=\u5206\u6790
-RepositoryNodeHelper.Reports=\u62A5\u8868
-RepositoryNodeHelper.Indicators=\u6307\u6807
```
Comment (scorreia) [inline i18n/plugins/org.talend.cwm.management.nl/src/main/java/org/talend/cwm/management/i18n/messages_zh_CN.properties]: is it expected that all these strings are removed?

File: i18n/plugins/org.talend.dataprofiler.core.nl/src/messages_el.properties [hunk 4]
```diff
@@ -196,18 +187,15 @@ CorePlugin.workspaceInUse=\u039F \u03A7\u03CE\u03C1\u03BF\u03C2 \u0395\u03C1\u03
...
 NewParserRulesWizard.defineProp=\u039F\u03C1\u03AF\u03C3\u03C4\u03B5 \u03C4\u03B9\u03C2 \u03B9\u03B4\u03B9\u03CC\u03C4\u03B7\u03C4\u03B5\u03C2
-ParserRuleMasterDetailsPage.add=\u03A0\u03C1\u03C3\u03B8\u03AE\u03BA\u03B7
 ParserRuleMasterDetailsPage.delete=\u0394\u03B9\u03B1\u03B3\u03C1\u03B1\u03C6\u03AE
 ExportParserRuleWizard.Error=\u03A3\u03C6\u03AC\u03BB\u03BC\u03B1
 ExportParserRuleWizardPage.browse=\u03A0\u03BB\u03BF\u03AE\u03B3\u03B7\u03C3\u03B7...
 ImportParserRuleWizard.Warning=\u03A0\u03C1\u03BF\u03B5\u03B9\u03B4\u03BF\u03C0\u03BF\u03AF\u03B7\u03C3\u03B7
 ImportParserRuleWizardPage.browsing=\u03A0\u03BB\u03BF\u03AE\u03B3\u03B7\u03C3\u03B7...
 PhoneNumberForm.options=\u0395\u03C0\u03B9\u03BB\u03BF\u03B3\u03AD\u03C2
 PhoneNumberForm.country=\u03A7\u03CE\u03C1\u03B1
-PhoneNumbStatisticsState.Count=\u039C\u03AD\u03C4\u03C1\u03B7\u03C3\u03B7
```
Comment (scorreia) [inline i18n/plugins/org.talend.dataprofiler.core.nl/src/messages_el.properties]: why are these strings removed?

File: i18n/plugins/org.talend.dataprofiler.core.nl/src/messages_en.properties [hunk 5]
```diff
@@ -352,7 +352,9 @@ DynamicIndicatorOptionsPage.indicatorSetting=Indicator settings
 UIMessages.lowerValuelessThanHigherValue=The lower value must be less than the higher value.
 BinsDesignerForm.numberOfBins=number of bins
 AbstractMetadataFormPage.status=Status\:
-RunAnalysisAction.failRunAnalysis=Fail to run the analysis: " {0} ".\nError message:\n{1}
+RunAnalysisAction.failRunAnalysis=Fail to run the analysis: " {0} ".
```
Comment (scorreia) [inline i18n/plugins/org.talend.dataprofiler.core.nl/src/messages_en.properties]: bad formatting here.

File: i18n/plugins/org.talend.dataprofiler.core.nl/src/messages_fr.properties [hunk 6]
```diff
@@ -352,7 +352,8 @@ DynamicIndicatorOptionsPage.indicatorSetting=Param
 UIMessages.lowerValuelessThanHigherValue=La plus petite valeur doit �tre strictement inf�rieure � la plus grande.
 BinsDesignerForm.numberOfBins=nombre d'intervalles
 AbstractMetadataFormPage.status=Statut \:
-RunAnalysisAction.failRunAnalysis=Echec lors de l'ex�cution de l'analyse: "{0}".  Message d'erreur:\n{1}
+RunAnalysisAction.failRunAnalysis=Echec lors de l'ex�cution de l'analyse: "{0}".  Message d'erreur:
```
Comment (scorreia) [inline i18n/plugins/org.talend.dataprofiler.core.nl/src/messages_fr.properties]: bad formatting here.

File: i18n/plugins/org.talend.dataprofiler.core.nl/src/messages_hr.properties [hunk 7]
```diff
@@ -26,7 +25,13 @@ DeleteSqlFileAction.delete=Obri\u0161i
 ColumnsComparisonMasterDetailsPage.sort=Sortiraj
 TextLengthForm.options=Opcije
 BasicThreePartURLSetupControl.url=URL
-ConnectionMasterDetailsPage.schema=\n\n	\n\nCollection\n\nSchema
+ConnectionMasterDetailsPage.schema=
```
Comment (scorreia) [inline i18n/plugins/org.talend.dataprofiler.core.nl/src/messages_hr.properties]: bad formatting here

File: i18n/plugins/org.talend.dataprofiler.core.nl/src/messages_hr.properties [hunk 8]
```diff
@@ -40,10 +45,15 @@ ColumnCorrelationNominalIntervalResultPage.Data=Podaci
 ColumnCorrelationNominalIntervalResultPage.Label=Labela
 ColumnCorrelationNominalIntervalResultPage.RowCount=Broj redaka
 ColumnCorrelationNominalIntervalResultPage.Row_Count=Broj redaka
-DQRuleMasterDetailsPage.add=Dodaj
 DQRuleMasterDetailsPage.delete=Obri\u0161i
 ImageLib.connection=Konekcija
-ImageLib.schema=\n\n	\n\nCollection\n\nSchema
+ImageLib.schema=
```
Comment (scorreia) [inline i18n/plugins/org.talend.dataprofiler.core.nl/src/messages_hr.properties]: bad formatting here

File: i18n/plugins/org.talend.dataprofiler.core.nl/src/messages_ja.properties [hunk 9]
```diff
@@ -482,8 +482,12 @@ ColumnCorrelationNominalIntervalResultPage.Distinct_Count=\u500B\u5225\u30AB\u30
 ColumnCorrelationNominalIntervalResultPage.DuplicateCount=\u91CD\u8907\u6570
 ColumnCorrelationNominalIntervalResultPage.Duplicate_Count=\u91CD\u8907\u6570
 ColumnCorrelationNominalIntervalResultPage.Label=\u30E9\u30D9\u30EB
-ColumnCorrelationNominalIntervalResultPage.RowCount=\u884C\u6570\n\n
-ColumnCorrelationNominalIntervalResultPage.Row_Count=\u884C\u6570\n\n
+ColumnCorrelationNominalIntervalResultPage.RowCount=\u884C\u6570
```
Comment (scorreia) [inline i18n/plugins/org.talend.dataprofiler.core.nl/src/messages_ja.properties]: bad formatting here

File: i18n/plugins/org.talend.cwm.management.nl/src/main/java/org/talend/cwm/management/i18n/messages_zh_CN.properties (hunk 3 above)
Comment (xqliu-talend) [reply i18n/plugins/org.talend.cwm.management.nl/src/main/java/org/talend/cwm/management/i18n/messages_zh_CN.properties]: some keys are still using in the studio, so should not remove them all

**Pull request merged**